docs/herramientas/explorador-de-bloques.md

```diff
@@ -44,7 +44,7 @@ https://explorer.eosdac.io
 
 ## Cypherblock
 
-https://www.cypherblock.io/
+https://www.blockcypher.com/
```

Comment:
I dont think this is the same project, lets just remove all mentions of cypherblock

Reply:
Fixed